lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_consular_cni_countries_when_residing_in_uk_or_ceremony_country.govspeak.erb

```diff
@@ -224,6 +229,50 @@
 
       %You can’t book an appointment to get advice at the embassy in person.%
 
+    <% elsif calculator.ceremony_country == 'bulgaria' &&
+      calculator.resident_of_ceremony_country? %>
+
+      Once the notary public has witnessed your documents, send the original documents to the British embassy in Sofia:
+
+      **British Embassy Sofia**
+
+      $A
+      Consular section
+      9, Moskovska St
+      1000 Sofia
+      Bulgaria
+      $A
+
+      There’s an additional fee for this - you must complete a [payment form](/government/publications/bulgaria-consular-fees) and email it to the British embassy in Sofia.
+
+      $C
+      <Consularsection.Sofia@fco.gov.uk>
+      $C
+
+      The embassy will contact you to confirm that the payment was accepted.
+
+      The embassy will display your notice of marriage publicly for 7 days before issuing your CNI.
+
+      You can collect the CNI from the embassy or ask for it to be sent by courier to an address in Bulgaria. You’ll need to pay any courier fees.
+
+      Make an appointment to collect your CNI at the embassy in Sofia.
+
+      ^You don’t need to stay in the country while your notice is posted.^
+
+      Once you have your CNI, you must contact the Bulgarian Ministry of Foreign Affairs to get it legalised (certified as genuine).
+
+      <%= render partial: 'bulgarian_foreign_affairs_address.govspeak.erb' %>
```

Comment:
I'd recommend we do as Chris suggests for now - I'd be surprised if leaving
the numbers out wasn't an oversight.

On Wed, 20 Jul 2016 at 17:09, Ikenna N. Okpala <notifications@github.com>
wrote:

> In
> lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_consular_cni_countries_when_residing_in_uk_or_ceremony_country.govspeak.erb
> <https://github.com/alphagov/smart-answers/pull/2646#discussion_r71555673>
> :
>
> > +      <Consularsection.Sofia@fco.gov.uk>
> > +      $C
> > +
> > +      The embassy will contact you to confirm that the payment was accepted.
> > +
> > +      The embassy will display your notice of marriage publicly for 7 days before issuing your CNI.
> > +
> > +      You can collect the CNI from the embassy or ask for it to be sent by courier to an address in Bulgaria. You’ll need to pay any courier fees.
> > +
> > +      Make an appointment to collect your CNI at the embassy in Sofia.
> > +
> > +      ^You don’t need to stay in the country while your notice is posted.^
> > +
> > +      Once you have your CNI, you must contact the Bulgarian Ministry of Foreign Affairs to get it legalised (certified as genuine).
> > +
> > +      <%= render partial: 'bulgarian_foreign_affairs_address.govspeak.erb' %>
>
> In the document it isn't defined.. I have highlighted and tagged you to
> the section
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/alphagov/smart-answers/pull/2646/files/5e9d86e16b8be71da783a8390f98700365013c3a..77ddf4480eacac25fbe51cff7a86803e2f7c98aa#r71555673>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ASR-5gYjWIDFf-rJJufxWbiEz50peTkvks5qXkhCgaJpZM4JQrLs>
> .
>